src/renderer/components/message/styles.module.scss

```diff
@@ -37,7 +37,8 @@ $reaction-transition-duration: 100ms;
 
 .vcard {
   display: flex;
-  cursor: pointer;
+  padding-left: 8px;
+  margin-bottom: 12px;
```

Comment:
the padding/margin here could be removed again when going for the general "padding: 9px;" suggestion above.

having simple, horizontally symmetric spacings  is also better wrt to the rtl issue that needs to be targeted at some point, btw